lib/models/BucketAzureInfo.ts

```diff
@@ -3,32 +3,34 @@ export type DeleteRetentionPolicy = {
     days: number;
 };
 
+export type AzureInfoMetadata = {
+    sku: string;
+    accessTier: string;
+    kind: string;
+    systemKeys: string[];
+    tenantKeys: string[];
+    subscriptionId: string;
+    resourceGroup: string;
+    deleteRetentionPolicy: DeleteRetentionPolicy;
+    managementPolicies: any[];
+    httpsOnly: boolean;
+    tags: any;
+    networkACL: any[];
+    cname: string;
+    azureFilesAADIntegration: boolean;
+    hnsEnabled: boolean;
+    logging: any;
+    hourMetrics: any;
+    minuteMetrics: any;
+    serviceVersion: string;
+};
+
 /**
  * Helper class to ease access to the Azure specific information for
  * storage accounts mapped to buckets.
  */
 export default class BucketAzureInfo {
-    _data: {
-        sku: string;
-        accessTier: string;
-        kind: string;
-        systemKeys: string[];
-        tenantKeys: string[];
-        subscriptionId: string;
-        resourceGroup: string;
-        deleteRetentionPolicy: DeleteRetentionPolicy;
-        managementPolicies: any[];
-        httpsOnly: boolean;
-        tags: any;
-        networkACL: any[];
-        cname: string;
-        azureFilesAADIntegration: boolean;
-        hnsEnabled: boolean;
-        logging: any;
-        hourMetrics: any;
-        minuteMetrics: any;
-        serviceVersion: string;
-    };
+    private data: AzureInfoMetadata;
```

Comment:
```suggestion
    private readonly data: AzureInfoMetadata;
```

And this type can be used in the constructor as well ?